website/source/docs/agent/configuration/server_join.html.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: "docs"
+page_title: "server_join Stanza - Agent Configuration"
+sidebar_current: "docs-agent-configuration-server_join"
+description: |-
+  The "server_join" stanza configures the Nomad agent to enable retry_join logic for connecting to Nomad servers.
```

Comment:
The server_join stanza specifies how the Nomad agent will discover and connect to Nomad servers.

Reply:
Below too